Dockerfile

```diff
@@ -50,37 +65,30 @@ ARG REACT_APP_RELEASE
 ARG REACT_APP_COMMITHASH
 ARG REACT_APP_BUILDTIME
 
-USER root
-RUN apt-install.sh build-essential
-
-# Use non-root user
-USER appuser
-
-# Install dependencies
-COPY --chown=appuser:appuser package*.json *yarn* /app/
-
-# Copy craco configuration
-COPY --chown=appuser:appuser craco.config.js /app/
```

Comment:
`craco.config.js` is not copied anymore, is it being used at all? Looking at the package.json there are scripts for craco:
```json
    "start": "craco start",
    "build": "craco build",
    "test": "craco test",
```
in addition to the react-scripts versions:
```json
    "start-react-scripts": "cross-env SASS_PATH=$(cd ./src/styles && pwd) react-scripts start",
    "build-react-scripts": "cross-env SASS_PATH=$(cd ./src/styles && pwd) react-scripts build",
    "test-react-scripts": "cross-env SASS_PATH=$(cd ./src/styles && pwd) react-scripts test",
```

If you think craco is not needed anymore then IMHO it'd be better to get rid of it completely than leave it as dead code.

Reply:
Well, good catch. I need to test that. We do want to use Craco just to get [SWC-tools](https://swc.rs/), but it was originally needed to configure SASS-plugins after a big node 20 dependency update.

The Craco should be replaced with a Vite framework at some point. The Vite is now already in use in Kukkuu-UI, because [the CRA is deprecated](https://github.com/reactjs/react.dev/pull/5487). More here: https://dev.to/ag2byte/create-react-app-is-officially-dead-h7o.

Reply:
Actually, after thinking this for a minute, the application would not even build without the craco-configuration, so the file must be included there. But, it should also be noted that the file is not available in the nginx-container, because the file is only needed in a build process, which is done in `staticbuilder`.

This can be tested by removing the `craco.config.js` and commanding `yarn build` -- an error will be thrown.